# Goal
float64[] goal_coords
float64 goal_z
float32 speed